src/routes/routes.js

```diff
@@ -216,10 +214,6 @@ export const modals = {
     isPrivate: false,
     forbiddenTokens: [],
   },
-  setPassword: {
```

Comment:
can you cross check if there such routes which are redundant?

Reply:
Yes, i couldnt find that such a route is invoked anywhere. Nor do we have such a component